services/test/services/errortelemetry.test.js

```diff
@@ -1,31 +1,63 @@
-const assert = require('assert');
+const fs             = require('fs');
+const assert         = require('assert');
+
 const app = require('../../src/app');
 
+const errorTelemetryService = 'telemetry/error';
+
 describe('\'ErrorTelemetry\' service', () => {
   it('registered the service', () => {
-    const service = app.service('errorTelemetry');
+    const service = app.service(errorTelemetryService);
 
     assert.ok(service, 'Registered the service');
   });
 });
 
 describe('Error Telemetry', () => {
   it('should fail with empty parameters', () => {
-    const service = app.service('errorTelemetry');
+    const service = app.service(errorTelemetryService);
     return service.create()
       .then(() => assert.fail('Should have failed to create()'))
       .catch((err) => assert.ok(err.message.match('^A data object must be provided')));
   });
 
+  it('missing fields should be rejected', async () => {
+    const service = app.service(errorTelemetryService);
+
+    const badQueries = [
+      'blah',
+      {'log':{'version': 1 }},
+      {'log':{'timestamp': 1526387942 }}
+    ];
+    for ( const badQuery in badQueries ) {
+      try {
+        await service.create(badQuery);
+        assert.fail('Should have failed above');
+      } catch (err) {
+        // expected
```

Comment:
There should be some assertion here into what kind of error `err` actually is. That way we won't receive false positives on type errors or others